docs/tools/tutorials/creating-users-and-groups.md

```diff
@@ -0,0 +1,145 @@
+
+## Pre-requisites
+For this tutorial, you need to deploy DataHub Quickstart and ingest sample data. 
+For detailed information, please refer to [Preparing Your Local DataHub Environment](/docs/tools/tutorials/references/prepare-datahub.md).
+
+:::note
+In this guide, ingesting sample data is optional.
+:::
+
+## Create Users And Groups With Python CLI
+
+You can ingest users and groups with `yaml` using Python CLI. 
+
+### Create User
+
+Save this `user.yaml` as a local file. 
+
+```yaml
+id: janedoe@acryl.io
+slack: @janedoe
+email: janedoe@acryl.io
+first_name: Jane
+last_name: Doe
+```
+
+Execute the following CLI command to ingest this user's information. 
+
+```
+datahub user create -c user.yaml
+```
+
+### Create Group
+
+Save this `group.yaml` as a local file. Note that the group includes a list of users who are admins (these will be marked as owners) and members.
+Within these lists, you can refer to the users by their ids or their urns, and can additionally specify their metadata inline within the group description itself. See the example below to understand how this works and feel free to make modifications to this file locally to see the effects of your changes in your local DataHub instance.
+
+```yaml
+id: engineering
+display_name: Engineering
+admins:
+  - "janedoe@acryl.io"
+members:
+  - "jane@acryl.io" # refer to a user either by id or by urn
+  - id: "joe@acryl.io" # inline specification of user
+    slack: "@joe_shmoe"
+    display_name: "Joe's Hub"
+```
+
+Execute the following CLI command to ingest this group's information. 
+
+```
+datahub group create -c group.yaml
+```
+
+
+## Create Users And Groups With Python SDK
+
+### Create User
+
+```python
+import logging
+
+from datahub.emitter.mce_builder import make_user_urn
+from datahub.emitter.mcp import MetadataChangeProposalWrapper
+from datahub.emitter.rest_emitter import DatahubRestEmitter
+
+# Imports for metadata model classes
+from datahub.metadata.schema_classes import CorpUserInfoClass
+
+log = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+user_urn = make_user_urn("janedoe")
+event: MetadataChangeProposalWrapper = MetadataChangeProposalWrapper(
+    entityUrn=user_urn,
+    aspect=CorpUserInfoClass(active=True,
+                             displayName="Jane Doe",
+                             email="janedoe@acryl.io",
+                             title="Software Engineer",
+                             firstName="Jane",
+                             lastName="Doe",
+                             fullName="Jane Doe"),
+)
+
+# Create rest emitter
+rest_emitter = DatahubRestEmitter(gms_server="http://localhost:8080")
+rest_emitter.emit(event)
+log.info(f"Created user {user_urn}")
+```
+
+This will create a user named `Jane Doe` with urn `urn:li:corpuser:janedoe`.
+
+### Create Group
+
+```python
+import logging
```

Comment:
unfortunately, group creation (specifically membership and ownership) is a bit more complex than this. Let's wait for #7595 to get in and then this code will become much simpler.